web/pages/speakers/[slug].tsx

```diff
@@ -60,8 +53,10 @@ const Speakers = ({
             <Heading>{name}</Heading>
             <div>{title}</div>
             <div>
-              <Link href={`/speakers/${name}`}>
-                <a>twitter.com/{twitter}</a>
+              <Link href={`https://twitter.com/${twitter}`}>
```

Comment:
Is there any reason to use `Link` for an external URL? Or could `href` instead be set on the `<a>` below?

Reply:
No, as far as I gather there's no reason to use `Link` for external links. _speakers/index.tsx_ already uses an `a`, thanks for catching this inconsistency.